linux.md

```diff
@@ -199,6 +199,7 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     sudo apt update
     sudo apt install postgresql-16
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
+    perl -i -pe "s/^logging_collector = on/logging_collector = off/" "$PGDATA/postgresql.conf"
```

Comment:
Use perl to set the `logging_collector = off` to enable psql message logging in terminal.
 `PGDATA` variable will be available once https://github.com/upleveled/system-setup/pull/80 is merged

Reply:
we probably don't need this for Linux or macOS

Reply:
Remove here
faf6debff142f9ef66ed3629c5f11d8f12bbd9fc